api/src/main/java/com/epam/pipeline/acl/cluster/ClusterApiService.java

```diff
@@ -112,4 +113,11 @@ public AllowedInstanceAndPriceTypes getAllowedInstanceAndPriceTypes(final Long t
     public List<NodeDisk> loadNodeDisks(final String name) {
         return nodeDiskManager.loadByNodeId(name);
     }
+
+
+    public InstanceDNSRecord changeInstanceDNSRecord(final Long regionId, final InstanceDNSRecord dnsRecord,
```

Comment:
I think that edge service is running under admin account, so we can add `@PreAuthorize(ADMIN_ONLY)` here